assignment2/company.rb

```diff
@@ -0,0 +1,95 @@
+require 'csv'
+require './employee.rb'
+
+module Designation
+    def self.included(klass)
+        klass.extend(ClassMethods)
+    end
+
+    module ClassMethods
+        def find_experience(experience)
+            if (experience > 5)
+                puts "Enployee Role: Team Lead"
+            else
+                puts "Employee Role: Developer"
+            end
+        end
+    end
+
+    def find_experience_obj(experience)
+        if (experience > 5)
+            puts "Enployee Role: Team Lead"
+        else
+            puts "Employee Role: Developer"
+        end
+    end
+end
+
+class Company
+    include Designation
+    def initialize(emp_data)
+        @emp_data = emp_data
```

Comment:
initialize method should get an array of employee objects, not employee data as in JSON.